backend/dataall/db/api/environment.py

```diff
@@ -622,29 +595,20 @@ def query_environment_invited_groups(
         return query
 
     @staticmethod
-    @has_resource_perm(permissions.LIST_ENVIRONMENT_GROUPS)
-    def paginated_environment_invited_groups(
-        session, username, groups, uri, data=None, check_perm=None
-    ) -> dict:
+    @has_resource_permission(permissions.LIST_ENVIRONMENT_GROUPS)
+    def paginated_environment_invited_groups(session, uri, data=None) -> dict:
         return paginate(
-            query=Environment.query_environment_invited_groups(
-                session, username, groups, uri, data
-            ),
+            query=Environment.query_environment_invited_groups(session, uri, data),
             page=data.get('page', 1),
             page_size=data.get('pageSize', 10),
         ).to_dict()
 
     @staticmethod
-    @has_resource_perm(permissions.LIST_ENVIRONMENT_GROUPS)
-    def list_environment_invited_groups(
-        session, username, groups, uri, data=None, check_perm=None
-    ) -> dict:
-        return Environment.query_environment_invited_groups(
-            session, username, groups, uri, data
-        ).all()
+    def list_environment_invited_groups(session, uri):
```

Comment:
permission decorator is missing from here

Reply:
This method is used while deploying env stack. We can't put a decorator, because we are out of user context